tier4_system_msgs/CMakeLists.txt

```
@@ -22,6 +22,7 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "msg/HazardStatusStamped.msg"
   "msg/ServiceLog.msg"
   "msg/IsAutonomousAvailable.msg"
+  "msg/MRMBehaviorStatus.msg"
```

Same as https://github.com/autowarefoundation/autoware_adapi_msgs/pull/16#discussion_r1000179751.

```suggestion
  "msg/MrmBehaviorStatus.msg"
```